### ISSUE TITLE
Inaccurate type declaration of Store#load

### ISSUE THREAD

**Issue by tpluscode** [oxigraph/oxigraph#955]
In `node.d.ts` I find

```js
/**
* @param {string} data
* @param {any} options
* @param {any} base_iri
* @param {any} to_graph_name
*/
load(data: string, options: any, base_iri: any, to_graph_name: any): void;
```

However, the readme shows a different interface (my rendition of `options`, assuming `Format` is a union type of allowed values but could be a simple string)

```js
load(data: string, options: {
  format: Format,
  base_iri?: string | NamedNode
  to_named_graph?: NamedNode | BlankNode | DefaultGraph
  unchecked?: boolean
  no_transaction?: boolean
})
```

**Comment by Tpt** [comment]
Hey! The documentation exposes the new function arguments with a single `data` field and the `options` dictionnary whereas the code and auto generated type script definition is is still compatible with the old argument lists (`data`, `type`, `base_iri`, `to_graph_name`). I plan to drop the old argument list in a future release but kept them for now to ease transition

**Comment by tpluscode** [comment]
Would it be possible actually express that accurately, without using `any` and with possible function overloads? Are types generated?

**Comment by Tpt** [comment]
They were auto generated in the latest apha release but #922 migrates to explicit types that removes all these `any`

**Comment by Tpt** [comment]
Released as part of 0.4.0-alpha.8
